app/javascript/components/pages/ApartmentIndex.test.js

```diff
@@ -0,0 +1,25 @@
+import React from "react"
+import "@testing-library/jest-dom"
+import {render, screen} from "@testing-library/react"
+import userEvent from "@testing-library/user-event"
+import { BrowserRouter } from "react-router-dom"
+import ApartmentIndex from "./ApartmentIndex"
+import { Card } from "reactstrap"
+
+describe("<ApartmentIndex />", () => {
+  render(
+    <BrowserRouter>
+      <ApartmentIndex />
+    </BrowserRouter>
+  )
+  it("renders without crashing", () => {
+    expect(screen.getByText(/apartment listings/i)).toBeInTheDocument();
+  })
+
+  it("renders Card",()=>{
+    render(
+      <Card/>
+    )
+   expect(Card).toBeDefined()
```

Comment:
What kind of content is being displayed here?  In other words, what information is being passed to your apartment index page?  You should also be checking iteration on data and check for a specific value in this test.